server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -99,3 +105,23 @@ def create_access_control_privilege_many(
             )
 
         audit_event_log.log_event()
+
+
+@router.get(
+    "/{application_id}/access_control_privileges",
+    response_model=List[schemas.FamAccessControlPrivilegeDto],
+    dependencies=[Depends(authorize_by_app_id)],  # only app admin can do this
+    description="Get Delegated Admin Privileges For an Application",
+)
+def get_access_control_privileges_by_application_id(
+    application_id: int,
+    token_claims: dict = Depends(jwt_validation.authorize),
+    access_control_privilege_service: AccessControlPrivilegeService = Depends(
+        access_control_privilege_service_instance
+    ),
+):
+    LOGGER.debug(
+        f"Get access control privileges for application: {application_id}, requestor: {token_claims}"
+    )
```

Comment:
We don't need this debug log I think. Also `requester: {token_claims}` does not match I think.